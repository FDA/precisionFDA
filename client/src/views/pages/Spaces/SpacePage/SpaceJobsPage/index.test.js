import React from 'react'
import { shallow } from 'enzyme'

import { SpaceJobsPage } from '.'


describe('SpaceJobsPage test', () => {
  it('should render', () => {
    const space = { isPrivate: true }
    const component = shallow(<SpaceJobsPage space={space} />)

    expect(component).toMatchSnapshot()
  })
})
