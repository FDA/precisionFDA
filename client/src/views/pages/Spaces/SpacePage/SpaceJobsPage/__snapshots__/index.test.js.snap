// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpaceJobsPage test should render 1`] = `
<Connect(SpaceLayout)
  space={
    Object {
      "isPrivate": true,
    }
  }
>
  <div
    className="space-page-layout__header-row"
  >
    <h2
      className="space-page-layout__sub-title"
    >
      Private Area Jobs
    </h2>
    <div
      className="space-page-layout__actions"
    />
  </div>
  <div
    className="pfda-padded-t20"
  >
    <Connect(SpaceJobsList) />
  </div>
  <AddDataModal
    loadDataHandler={[Function]}
    space={
      Object {
        "isPrivate": true,
      }
    }
  />
</Connect(SpaceLayout)>
`;
